CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc

```diff
@@ -235,8 +235,8 @@ void TotemDAQMappingESSourceXML::setIntervalFor(const edm::eventsetup::EventSetu
 
     // event id "1:min" has a special meaning and is translated to a truly minimal event id (1:0:0)
     EventID startEventID = bl.validityRange.startEventID();
-    if (startEventID == EventID(1, 0, 1))
-      startEventID = EventID(1, 0, 0);
+    if (startEventID.event() == 1)
+      startEventID = EventID(startEventID.run(), startEventID.luminosityBlock(), 0);
```

Comment:
I suspect that "1" in "1:min" was not special and the comment should be 
```      // event id "N:min" has a special meaning and is translated to a truly minimal event id (N:0:0)  ```
We have clearly discovered in MC that the first call is made not inside an event but inside a beginRun transition to make this comment for MC. This PR is apparently the first time we tried this code on the transition run.

If you look in the job failing in 283820 the file first entry is the run transition
```
         283820              0              0              0 (Run)
```